TestProjects/NotificationSamples/Assets/Scripts/AndroidTest.cs

```diff
@@ -0,0 +1,262 @@
+﻿using System;
+using System.Reflection;
+using System.Collections;
+using System.Collections.Generic;
+using System.Collections.Specialized;
+using UnityEngine;
+using UnityEngine.UI;
+
+#if PLATFORM_ANDROID
+using Unity.Notifications.Android;
+#endif
+
+public class AndroidTest : MonoBehaviour
+{
+    public List<AndroidNotificationTemplate> androidNotifications;
+
+#if PLATFORM_ANDROID
+
+    private GameObjectReferences gameObjectReferences;
+    private Dictionary<string, OrderedDictionary> groups;
+    private Logger LOGGER;
```

Comment:
Private fields should start with `m_`.

Reply:
fixed these, and some others I found, hope I found them all